### PR TITLE
Add pipeline with MSVC compiler

### PR DESCRIPTION
**Pull request by fabinsch**
Run our CI with the default MSVC compiler as requested in #38 after the PR #33.

### CODE REVIEW

**Review by jcarpent — DISMISSED** [review]

**Comment by fabinsch** [comment]
hey @ahoarau , in our CI we are using [windows-2019](https://github.com/Simple-Robotics/proxsuite/pull/49/files#diff-8f553a73b02913f93ae446780b2e19f0998f0509df5cefa734d687b054e60106R23) (with clang) and [windows-latest](https://github.com/Simple-Robotics/proxsuite/pull/49/files#diff-8f553a73b02913f93ae446780b2e19f0998f0509df5cefa734d687b054e60106R26) with MSVC, which is failing currently.

**Review by jcarpent — APPROVED** [review]

**Comment by fabinsch** [comment]
note, I just activated again the pipelines including the the self-hosted-arm64 runner. That's why we still have 11 expected checks.

File: include/proxsuite/proxqp/utils/random_qp_problems.hpp [hunk 1]
```diff
@@ -65,6 +66,42 @@ namespace rand {
 
 using proxqp::u32;
 using proxqp::u64;
+
+#if _MSC_VER
```
Comment (jcarpent) [inline include/proxsuite/proxqp/utils/random_qp_problems.hpp:70]:
```suggestion
#ifdef _MSC_VER
```

Comment (fabinsch) [reply include/proxsuite/proxqp/utils/random_qp_problems.hpp]: thank you

**Review by jcarpent — APPROVED** [review]

**Pull request merged**